pymc_marketing/clv/utils.py

```diff
@@ -42,42 +42,37 @@ def to_xarray(customer_id, *arrays, dim: str = "customer_id"):
 
 def customer_lifetime_value(
     transaction_model,
-    customer_id: pd.Series | np.ndarray,
-    frequency: pd.Series | np.ndarray,
-    recency: pd.Series | np.ndarray,
-    T: pd.Series | np.ndarray,
+    transaction_data: pd.DataFrame,
```

Comment:
Why does this not include the "monetary_value" column like the other methods? Then could remove the monetary_value parameter. 
Could it be different dimensions?

Reply:
The idea behind separate `GammaGammaModel.expected_customer_lifetime_value` and `utils.customer_lifetime_value` functions is that users can use the latter if they wish to use an external monetary value model, whereas the former performs both predictions and lifetime value projections. However, I do agree a separate `monetary_value` parameter creates potential for dimensionality errors.

Reply:
Would broadcasting be supported if the correct `transaction_data` or `monetary_value` were provided? Or is broadcast support part of your long term vision of the functionality?
If not, combining would provide some consistency